tf-psa-crypto/drivers/builtin/src/sha256.c

```diff
@@ -414,13 +414,6 @@ static size_t mbedtls_internal_sha256_process_many_a64_crypto(
     return processed;
 }
 
-#if defined(MBEDTLS_SHA256_USE_ARMV8_A_CRYPTO_IF_PRESENT)
-/*
- * This function is for internal use only if we are building both C and Armv8-A
- * versions, otherwise it is renamed to be the public mbedtls_internal_sha256_process()
- */
-static
-#endif
 int mbedtls_internal_sha256_process_a64_crypto(mbedtls_sha256_context *ctx,
```

Comment:
More precisely, we should remove the code that makes this function non-static in some cases  thus always have the `static`
```suggestion
static int mbedtls_internal_sha256_process_a64_crypto(mbedtls_sha256_context *ctx,
```